ui/page/privacy/setup_mixer_accounts_page.go

```diff
@@ -62,162 +52,150 @@ func NewSetupMixerAccountsPage(l *load.Load, dcrWallet *dcr.Asset) *SetupMixerAc
-													<b>Unmixed </b> account will be the change handling account.
-												</span>`
-
-															return layout.Inset{
-																Left: values.MarginPadding8,
-															}.Layout(gtx, renderers.RenderHTML(txt3, pg.Theme).Layout)
-														}),
-													)
-												})
-											}),
-										)
-									})
-								}),
-							)
-						}),
-						layout.Rigid(func(gtx C) D {
-							gtx.Constraints.Min.X = gtx.Constraints.Max.X
-							return pg.autoSetupClickable.Layout(gtx, pg.autoSetupLayout)
-						}),
-						layout.Rigid(func(gtx C) D {
-							gtx.Constraints.Min.X = gtx.Constraints.Max.X
-							return pg.manualSetupClickable.Layout(gtx, pg.manualSetupLayout)
-						}),
-					)
-				})
-			},
-		}
-		return page.Layout(pg.ParentWindow(), gtx)
-	}
-
-	return cryptomaterial.UniformPadding(gtx, body)
+func (pg *SetupMixerAccountsPage) Layout(gtx C) D {
+	return pg.Theme.Card().Layout(gtx, func(gtx C) D {
+		gtx.Constraints.Min.X = gtx.Constraints.Max.X
+		return layout.Inset{Top: values.MarginPadding25}.Layout(gtx, func(gtx C) D {
+			return layout.Flex{Axis: layout.Vertical, Alignment: layout.Start}.Layout(gtx,
+				layout.Rigid(func(gtx C) D {
+					label := pg.Theme.H6(values.String(values.StrSetUpStakeShuffleAutoOrManualA))
+					return layout.Inset{
+						Left: values.MarginPadding24,
+					}.Layout(gtx, label.Layout)
+				}),
+				layout.Rigid(func(gtx C) D {
+					label := pg.Theme.Body1(values.String(values.StrSetUpStakeShuffleAutoOrManualB))
+					return layout.Inset{
+						Left: values.MarginPadding24,
+					}.Layout(gtx, label.Layout)
+				}),
+				layout.Rigid(func(gtx C) D {
+					// TODO: Find a way to make Mixed and Unmixed bold while keeping to the theme.
+					label := pg.Theme.Body1(values.String(values.StrSetUpStakeShuffleAutoOrManualC))
+					return layout.Inset{
+						Top:  values.MarginPadding10,
+						Left: values.MarginPadding24,
+					}.Layout(gtx, label.Layout)
+				}),
+				layout.Rigid(func(gtx C) D {
+					label := pg.Theme.Body1(values.String(values.StrSetUpStakeShuffleAutoOrManualD))
+					return layout.Inset{
+						Top:  values.MarginPadding10,
+						Left: values.MarginPadding24,
+					}.Layout(gtx, label.Layout)
+				}),
+				layout.Rigid(func(gtx C) D {
+					return layout.Spacer{Height: values.MarginPadding80}.Layout(gtx)
+				}),
+				layout.Rigid(func(gtx C) D {
+					line := pg.Theme.Separator()
+					return layout.Inset{
+						Top:   values.MarginPadding10,
+						Left:  values.MarginPadding24,
+						Right: values.MarginPadding24,
+					}.Layout(gtx, line.Layout)
+				}),
+				layout.Rigid(func(gtx C) D {
+					return pg.autoSetupClickable.Layout(gtx, pg.autoSetupLayout)
+				}),
+				layout.Rigid(func(gtx C) D {
+					return pg.manualSetupClickable.Layout(gtx, pg.manualSetupLayout)
+				}),
+				layout.Rigid(func(gtx C) D {
+					return layout.Spacer{Height: values.MarginPadding10}.Layout(gtx)
+				}),
+			)
+		})
```

Comment:
too many repeated Insets

you can try:

```
return layout.Inset{
			Top: values.MarginPadding25,
			Left: values.MarginPadding24,
			Right: values.MarginPadding24,
			}.Layout(gtx, func(gtx C) D {...
```

Reply:
Sure. Prevents the buttons from covering the whole width of the screen but I guess thats fine.